src/modules/commands/discord/urbandictionary.ts

```diff
@@ -4,6 +4,7 @@ import { isRecord } from '@app/validators/object'
 import camelcaseKeys from 'camelcase-keys'
 import discord from 'discord.js'
 import { constants } from '@app/global/constants'
+import { mod } from 'mathjs'
```

Comment:
Hmm, this doesn't look like `@app/utils/math` 🤔

Reply:
lol I trusted VS code and didn't look back